src/pages/basic_data/app_admin/CustomerInstitutionMetadataForm.tsx

```diff
@@ -43,6 +46,7 @@ export const CustomerInstitutionMetadataForm = ({
   const { t } = useTranslation();
   const history = useHistory();
   const dispatch = useDispatch();
+  const [customerData, setCustomerData] = useState<Organization | null>(null);
```

Comment:
2 ting:
- customer og organization er to ulike ting for oss, hvor org er fra cristin og customer er fra NVA. Så blir litt forvirra når jeg ser at `customerData` er av typen `Organization` 🤔 
- hver setState trigger (minst) en ny re-render, og ser ikke noen åpenbar grunn til å ha en ekstra flyt for å håndtere den her, i tillegg til at den settes i state i formik. Men må grave litt mer i akkurat den merker jeg.. mulig det var gjort for å unngå ekstra api-kall, men det problemet minskes iaf med konsistent nøkkel til react-query

```suggestion
  const [organizationData, setOrganizationData] = useState<Organization | null>(null);
```